docs/guide/explanation/the-deployment-process.mdx

```diff
@@ -0,0 +1,53 @@
+# The deployment process
+Deployment begins via one of Zephyr Cloud's bundler plugins.
+When added to your bundler's plugin list, the plugin deploys your bundled applicatioon to a remote Zephyr Cloud instance.
+
+## Deployment lifecycle
+The deployment lifecycle can be split into three stages:
+
+*   [Initial bundling](#initial-bundling)
+*   [Fact gathering](#fact-gathering)
+*   [Snapshot upload and tagging](#snapshot-upload-and-tagging)
+
+### Initial bundling
+The application is bundled according to your bundler's configuration.
+The Zephyr Cloud bundler plugin runs as the last step in the bundling process, initiating the next stage of the process.
+
+### Fact gathering
+The bundler plugin first attempts to gather facts about the application.
+Deployments are identified by metadata relating to the application, including:
+
+*   Details extracted to the Git repository's `origin` remote URL:
+    *   The username/organization
+    *   The repository name
+*   The current Git branch
+*   [Whether the application has Module Federation enabled](#module-federation-and-micro-frontends)
+
+Once the details have been gathered, the plugin establishes a connection to the remote Zephyr Cloud instance.
+If not already authenticated, the plugin opens a browser window to authenticate.
+Otherwise, the plugin uses previously stored credentials located in `~/.zephyr` to authenticate.
```

Comment:
I'm not sure we want to share this in this fashion. Perhaps the location is more of a note rather than raw comment like this.

Reply:
The `~/.zephyr` (or this level of details in general) should only be in Error code to resolve errors, content of this folder or how we handle user's local cache might change in the future

Reply:
Removed this note in [4b97ebc](https://github.com/ZephyrCloudIO/zephyr-documentation/pull/5/commits/4b97ebc4b5a7faa4c146688380a4e6dc4b217796)